config/integration/apps.yml

```diff
@@ -51,13 +51,7 @@ captain:
       "type": "text",
       "name": "assistant_id",
       "validation": "required",
-    },
-    {
-      "label": "Inbox Ids",
-      "type": "text",
-      "name": "inbox_ids",
-      "validation": "",
-    },
+    }
```

Comment:
@sojan-official removed this from here, handling it on the the frontend directly